src/test/run-make-fulldeps/instrument-coverage/filecheck-patterns.txt

```diff
@@ -0,0 +1,32 @@
+# Check for metadata, variables, declarations, and function definitions injected
+# into LLVM IR when compiling with -Zinstrument-coverage.
```

Comment:
I was also concerned about brittleness actually. I made changes to remove more of the more complex type specifiers, just in case, and I took advantage of your suggestion to use CHECK-NEXT, so I can focus the checks on the specific things I believe are stable and required.